tdrs-frontend/nginx/nginx.conf

```diff
@@ -1,15 +1,27 @@
-server {
-  listen 80;
 
-  location / {
-    root /usr/share/nginx/html;
-    index index.html index.htm;
-    try_files $uri $uri/ /index.html;
-  }
+worker_processes 1;
+daemon off;
 
-  error_page 500 502 503 504 /50x.html;
+error_log stderr;
+events { worker_connections 1024; }
 
-  location = /50x.html {
-    root /usr/share/nginx/html;
-  }
+http {
+    charset utf-8;
+    log_format cloudfoundry 'NginxLog "$request" $status $body_bytes_sent';
+    access_log /dev/stdout cloudfoundry;
+    default_type application/octet-stream;
+    include mime.types;
+    sendfile on;
+
+    tcp_nopush on;
+    keepalive_timeout 30;
+    port_in_redirect off; # Ensure that redirects don't include the internal container PORT - 8080
+
+    server {
```

Comment:
Nginx build pack requires us to have this templated port variable. We don't get to choose what port this runs on.